WordPress/AbstractFunctionParameterSniff.php

```diff
@@ -77,6 +78,44 @@ public function process_matched_token( $stackPtr, $group_name, $matched_content
 		}
 	}
 
+	/**
+	 * Verify if the current token is a function call. Behaves like the parent method, except that
+	 * it returns false if there is no opening parenthesis after the function name (likely a
+	 * function import) or if it is a first class callable.
+	 *
+	 * @param int $stackPtr The position of the current token in the stack.
+	 *
+	 * @return bool
+	 */
+	public function is_targetted_token( $stackPtr ) {
+		if ( ! parent::is_targetted_token( $stackPtr ) ) {
+			return false;
+		}
+
+		$next = $this->phpcsFile->findNext( Tokens::$emptyTokens, ( $stackPtr + 1 ), null, true );
+
+		if ( \T_OPEN_PARENTHESIS !== $this->tokens[ $next ]['code'] ) {
+			// Not a function call (likely a function import).
+			return false;
+		}
+
+		if ( isset( $this->tokens[ $next ]['parenthesis_closer'] ) === false ) {
```

Comment:
Out of curiosity, why not write this as

```suggestion
		if ( !isset( $this->tokens[ $next ]['parenthesis_closer'] ) ) {
```

?

Also, if we need to be explicit, then, following the rest of the codebase (not saying I'm agreeing with this rule), we should write it in Yoda style

```suggestion
		if ( false === isset( $this->tokens[ $next ]['parenthesis_closer'] ) ) {
```

no?

Reply:
@dingo-d Yoda is not necessary here either way as the first part of the condition is not a variable.

Reply:
Ok, but still, we could use `!isset` here, it's perfectly valid 🤷🏼‍♂️

Reply:
Could, but IMO not a blocker for this PR. To be honest, I find the `!` is easily overlooked, so having the explicit comparison makes the code more readable.